metrics/prometheus.go

```diff
@@ -533,10 +541,34 @@ func (c *PrometheusCollector) Collect(ch chan<- prometheus.Metric) {
 }
 
 const (
-	containerLabelPrefix = "container_label_"
-	containerEnvPrefix   = "container_env_"
+	// ContainerLabelPrefix is the prefix added to all container labels added
+	// to exported metrics.
+	ContainerLabelPrefix = "container_label_"
+	// ContainerEnvPrefix is the prefix added to all env variables added to
+	// exported metrics.
+	ContainerEnvPrefix = "container_env_"
 )
 
+// DefaultContainerLabels implements ContainerLabelsFunc. It exports the
+// container name, first alias, image name as well as all its env and label
+// values.
+func DefaultContainerLabels(container *info.ContainerInfo) map[string]string {
+	set := map[string]string{"id": container.Name}
+	if len(container.Aliases) > 0 {
+		set["name"] = container.Aliases[0]
+	}
+	if image := container.Spec.Image; len(image) > 0 {
+		set["image"] = image
+	}
```

Comment:
Should we export these intermediate steps for reuse? Or at least make the keys (`image`, `name`, `id`) exported?